edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/CombinedKubernetesConfig.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.Azure.Devices.Edge.Agent.Kubernetes
+{
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Azure.Devices.Edge.Util.Json;
+    using Newtonsoft.Json;
+
+    public class CombinedKubernetesConfig
```

Comment:
Should we use CombinedKubernetesConfig to more accurately represent the state of the pod, i.e. having two images - proxy + module? Currently, there appears to be just one image name and create pod parameters, when a pod really has two images.

Reply:
I like this idea. I created a separate PBI about it